aea/aea.py

```diff
@@ -114,8 +126,10 @@ def handle(self, envelope: Envelope) -> None:
         """
         protocol = self.resources.protocol_registry.fetch(envelope.protocol_id)
 
-        error_handler = self.resources.handler_registry.fetch_by_skill("default", "error")
-        assert error_handler is not None, "ErrorHandler not initialized"
+        handlers = self.resources.handler_registry.fetch_by_skill("default", "error")
```

Comment:
I am not sure why you amended this @marcofavorito . It should not be possible for more than one to be returned (we only allow unique handlers per protocol in a skill and only unique skills).

Reply:
My change is wrong, thanks for spotting it.